src/app/myOrders/orders/js/orders.service.js

```diff
@@ -37,7 +37,10 @@ function ocOrdersService($filter, OrderCloudSDK){
         }
 
         if(parameters.status){
-            angular.extend(parameters.filters, {Status: parameters.status});
+            angular.extend(parameters.filters, {status: parameters.status});
+        } else {
+            //TODO: replace with "!Unsubmitted" when ! operator is fixed in API EX-1166
+            angular.extend(parameters.filters, {status: 'Open|AwaitingApproval|Completed|Declined|Cancelled'});
```

Comment:
I was working on a similar issue (F51-135). you're still missing status in route params so the status filters won't work at all. 

Do you want to just delete your changes? or add the status parameter?